src/gui/components/navbar/navbar.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import container, { title } from './navbar.styles'
+
 const Navbar = ({ style, className }) => {
   return (
-    <div style={style} className={className}>
-      Workbench
+    <div style={style} className={`${container} ${className}`}>
```

Comment:
Ok I prefer the `div className={css(...)} />` version then.
It seems to scale better (related to https://github.com/Zenika/react-workbench/pull/89#discussion_r125825015)